Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN go install github.com/vektra/mockery/v2@v2.52.2
 RUN go version
 
 # Install javascript dependencies
-RUN npm install -g @openapitools/openapi-generator-cli@2.13.4
+RUN npm install -g @openapitools/openapi-generator-cli@2.21.0
```

Comment:
Can you not read ther docs? Pushing this isn't testing as it will affect all package generation